matrix_benchmarking/upload_lts.py

```diff
@@ -0,0 +1,72 @@
+import logging
+import getpass
+import datetime as dt
+import requests
+import json
+
+import matrix_benchmarking.common as common
+import matrix_benchmarking.cli_args as cli_args
+import matrix_benchmarking.store as store
+
+def main(
+        workload: str = "",
+        results_dirname: str = "",
+        filters: list[str] = [],
+        horreum_url: str = "",
+        keycloak_url: str = "",
+        horreum_test: str = "",
+        horreum_uname: str = "",
+        horreum_passwd: str = "",
```

Comment:
Expanding on this one.

We do have `.env` functionality already in the codebase.

Biggest issue I'm now seeing is that when we don't specify the correct args, the entirety of the `kwargs` is printed out to console, so if we expand out the functionality and forget to update a CI, we may leak credentials...  Currently working on figuring this one out

Reply:
hum, I see. I didn't have credentials in mind when I coded these bits!
I let you try to find a way, even hacky, like having another file `.env.secret` that you parse independently of the other args. What ever works and can't leak the passwords by mistake will fit :)

Reply:
I've figured out a solution, since kwargs only gets updated, another kwargs dict was made for sensitive data.  A boolean was added to `check_mandatory_values` to prevent printing sensitive data to console.

Reply:
This approach also keeps the arguments out of the command line, while still allowing for enviornment variables (via OS or `.env`)